### ISSUE TITLE
Support for libraries in :local/root dependencies

### ISSUE THREAD

**Issue by bowbahdoe** [liquidz/antq#115]
Right now antq doesn't report on libraries that are included in `:local/root` subprojects

**Comment by liquidz** [comment]
@bowbahdoe Sorry for late action, but I just tried to implement in [feature/local-root](https://github.com/liquidz/antq/tree/feature/local-root) branch.
Could you try?

e.g.
```shell
$ cat deps.edn
{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
        local/root {:local/root "./foo"}}}

$ cat foo/deps.edn
{:deps {medley/medley {:mvn/version "1.2.0"}}}

$ clojure -Sdeps '{:deps {com.github.liquidz/antq {:git/sha "9b52adbb213bd0adff5547b1e27f8625e20bbc08"}}}' -M -m antq.core
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

|        :file |               :name | :current | :latest |
|--------------+---------------------+----------+---------|
|     deps.edn | org.clojure/clojure |   1.10.0 |  1.10.3 |
| foo/deps.edn |       medley/medley |    1.2.0 |   1.3.0 |

Available diffs:
- https://github.com/clojure/clojure/compare/clojure-1.10.0...clojure-1.10.3
- https://github.com/weavejester/medley/compare/1.2.0...1.3.0
```

**Comment by bowbahdoe** [comment]
I have a bit of a pathological example that probably doesn't represent 99% of reasonable projects, but:

![image](https://user-images.githubusercontent.com/5004262/140683200-4072ba6b-ad4b-4539-8da0-b2fed58ed0e4.png)

**Comment by liquidz** [comment]
@bowbahdoe Thanks for your trial!
Is that working fine for you?

**Comment by bowbahdoe** [comment]
There is some weirdness like the duplicated deps, but for my purposes that it works for my purpose

**Comment by liquidz** [comment]
@bowbahdoe Thanks!

Do you have duplicate definitions for multiple aliases, etc.?
If not, it's a bug, but I can't reproduce it in my environment...

**Comment by bowbahdoe** [comment]
Yep I do.

Again, super pathological but i'm not a big fan of override-deps as of yet

```
{:paths   ["."
           "components/log/src"
           "components/log/resources"
           "components/file-store/src"
           "components/file-store/resources"
           "bases/mtgbot/src"
           "bases/mtgbot/resources"
           "bases/home_server/src"
           "bases/home_server/resources"]
 :aliases {:dev   {:extra-paths ["." "development/src"]
                   :extra-deps  {libraries/clojure          {:local/root "libraries/clojure"}
                                 libraries/tools-deps-alpha {:local/root "libraries/tools-deps-alpha"}
                                 libraries/build-clj        {:local/root "libraries/build-clj"}}}

           :build {:deps       {libraries/tools-deps-alpha {:local/root "libraries/tools-deps-alpha"}
                                libraries/build-clj        {:local/root "libraries/build-clj"}}
                   :paths      ["build/resources"]
                   :ns-default build}

           :test  {:extra-paths ["components/log/test"
                                 "components/file-store/test"
                                 "bases/mtgbot/test"
                                 "bases/home_server/test"]}

           :antq  {:main-opts  ["-m" "antq.core"]
                   :extra-deps {libraries/antq {:local/root "libraries/antq"}}}}

 :deps    {libraries/clojure         {:local/root "libraries/clojure"}
           libraries/mulog           {:local/root "libraries/mulog"}
           libraries/data-json       {:local/root "libraries/data-json"}
           libraries/hiccup          {:local/root "libraries/hiccup"}
           libraries/ring            {:local/root "libraries/ring"}
           libraries/discord4j       {:local/root "libraries/discord4j"}
           libraries/data-xml        {:local/root "libraries/data-xml"}
           libraries/aws-java-sdk-s3 {:local/root "libraries/aws-java-sdk-s3"}
           libraries/cats            {:local/root "libraries/cats"}
           libraries/antq            {:local/root "libraries/antq"}
           libraries/sqlite-jdbc     {:local/root "libraries/sqlite-jdbc"}
           libraries/next-jdbc       {:local/root "libraries/next-jdbc"}
           libraries/tools-logging   {:local/root "libraries/tools-logging"}
           libraries/logback-classic {:local/root "libraries/logback-classic"}
           libraries/timbre          {:local/root "libraries/timbre"}
           libraries/core-async      {:local/root "libraries/core-async"}
           libraries/promesa         {:local/root "libraries/promesa"}
           libraries/reitit          {:local/root "libraries/reitit"}}}
```

libraries/antq

```
{:deps {com.github.liquidz/antq {:git/sha "9b52adbb213bd0adff5547b1e27f8625e20bbc08"}}}
```

**Comment by liquidz** [comment]
@bowbahdoe Ah, I got it. Thanks!
Fixed in d6527b9.

**Comment by liquidz** [comment]
Now merged to `dev` branch.

**Comment by liquidz** [comment]
I just released v1.3.0 #122
